neps/nep-0568.md

```diff
@@ -163,6 +163,16 @@ supporting smooth transitions without altering storage structures directly.
 
 ### Stateless Validation
 
+### State Witness
```

Comment:
Can you add a quick preface with very high level description? It feels like you're jumping right into the nitty gritty details.